src/segmentation_a_direct.jl

```diff
@@ -112,7 +112,9 @@ Compute the gradient magnitude of an image using the Sobel operator.
 """
 function imgradientmag(img)
     h = centered([-1 0 1; -2 0 2; -1 0 1]')
```

Comment:
Is this a Sobel kernel? If so maybe just add a comment after to indicate so

Reply:
Yes, I noted that in the docstring just a couple of lines earlier.